backend/src/graphql/resolver/AdminResolver.ts

```diff
@@ -460,6 +466,99 @@ export class AdminResolver {
       linkList: transactionLinks.map((tl) => new TransactionLink(tl, new User(user))),
     }
   }
+
+  @Authorized([RIGHTS.CREATE_CONTRIBUTION_LINK])
+  @Mutation(() => ContributionLink)
+  async createContributionLink(
+    @Args()
+    {
+      amount,
+      name,
+      memo,
+      cycle,
+      validFrom,
+      validTo,
+      maxAmountPerMonth,
+      maxPerCycle,
+    }: ContributionLinkArgs,
+  ): Promise<ContributionLink> {
```

Comment:
Added some validations and log-statements, as prepared on my local branch.
How can we introduce them in this branch?
Should i manually add them or do you want to review them before on an other way?